### ISSUE TITLE
Patch for fixing CPU throttling issues from CFS quotas

### ISSUE THREAD

**Issue by tbarrella** [awslabs/amazon-eks-ami#372]
Hello, maybe this is something that will happen naturally at some point, but it's pretty desired, so I wanted to ask about it/track it...

**What would you like to be added**:
Applying the [kernel patch](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/kernel/sched/fair.c?id=de53fd7aedb100f03e5d2231cfce0e4993282425) discussed in https://github.com/kubernetes/kubernetes/issues/67577 to EKS AMIs

**Why is this needed**:
To significantly improve performance for some workloads without having to disable CFS quotas (see the issue and patch)

**Comment by csko** [comment]
Related issue:
https://github.com/aws/containers-roadmap/issues/175

Seems like the patch is now backported so we can use it by switching to v4.14.154, v4.19.84, or 5.4+.

**Comment by mogren** [comment]
The latest release, [v20191119](https://github.com/awslabs/amazon-eks-ami/releases/tag/v20191119), is using the `4.14.152-127.182.amzn2` kernel.

**Comment by hugoprudente** [comment]
Amazon Linux 2 just had rolled out the new kernel version 4.14.154 is available for usage.

```
[ec2-user@ip-172-31-38-202 ~]$ uname -a
Linux ip-172-31-38-202.eu-west-1.compute.internal 4.14.154-128.181.amzn2.x86_64 #1 SMP Sat Nov 16 21:49:00 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by M00nF1sh** [comment]
the v20191213 amis are already using 4.14.154-128.181.amzn2.x86_64
https://github.com/awslabs/amazon-eks-ami/releases/tag/v20191213

**Comment by JacobHenner** [comment]
Is this patch included in 5.10.35-31.135.amzn2.x86_64?
